src/views/ProjectSelectionView.vue

```diff
@@ -5,34 +5,38 @@ import ProjectSelectionTable from '@/components/ProjectSelectionTable.vue';
 import Button from 'primevue/button';
 import Dialog from 'primevue/dialog';
 
-    const display = ref(false);
+const display = ref(false);
 
-    const open = () => {
-      display.value = true;
-    };
-
-    const close = () => {
-      display.value = false;
-    };
+const open = () => {
+  display.value = true;
+};
 
+const close = () => {
+  display.value = false;
+};
 </script>
 
 <template>
   <div class="grid">
     <div class="col-10">
       <div class="card">
-        <h5>Übersicht der Liegenschaften</h5>
+        <h5>{{ $t('projectSelection.title') }}</h5>
         <ProjectSelectionTable />
       </div>
     </div>
     <div class="col-2">
-      <Button label="Liegenschaft hinzufügen" icon="pi pi-plus" style="width: auto" @click="open" />
+      <Button
+        label="$t('projectSelection.add')"
+        icon="pi pi-plus"
+        style="width: auto"
+        @click="open"
+      />
       <Dialog
-          header="Liegenschaft hinzufügen"
-          v-model:visible="display"
-          :breakpoints="{ '960px': '75vw' }"
-          :style="{ width: '30vw' }"
-          :modal="true"
+        header="Liegenschaft hinzufügen"
```

Comment:
missing translation.
maybe one could use the same key as above: $t('projectSelection.add')